tests/ert/unit_tests/scheduler/test_slurm_driver.py

```diff
@@ -450,3 +459,26 @@ async def finished(iens: int, returncode: int):
     assert not Path(
         "survived"
     ).exists(), "The process children of the job should also have been killed"
+
+
```

Comment:
Would be nice to also test that we can obtain a zero exit code as well, parametrize with `[("true", 0), )("false", 1)]` ?